components/taggroup/stories/taggroup.stories.js

```diff
@@ -61,66 +120,196 @@ export default {
 		},
 		packageJson,
 		metadata,
-	},
-};
-
-export const Default = TagGroups.bind({});
-Default.args = {
-	ariaLabel: "Tags",
-	items: [
-		{
-			label: "Tag 1",
+		downState: {
+			selectors: [".spectrum-Tag", ".spectrum-ActionButton"],
 		},
-		{
-			label: "Tag 2",
-		},
-		{
-			label: "Tag 3",
+		status: {
+			type: "migrated",
 		},
+	},
+	decorators: [
+		withDownStateDimensionCapture,
 	],
+	tags: ["migrated"],
 };
 
+export const Default = TagGroups.bind({});
+Default.tags = ["!autodocs"];
+
 // ********* DOCS ONLY ********* //
+/**
+ * A tag group on its own should always have a label. Labels can be placed either on top or on the side of the tags, but top labels are the default and are recommended because they work better with long copy, localization, and responsive layouts.
+ */
+export const DefaultWithLabel = TagGroups.bind({});
+DefaultWithLabel.storyName = "Label position - default/top";
+DefaultWithLabel.tags = ["!dev"];
+DefaultWithLabel.parameters = {
+	chromatic: {
+		disableSnapshot: true,
+	},
+};
+DefaultWithLabel.args = {
+	actionButtonText: "",
+	helpText: "",
+	items: exampleTagItems,
+	fieldLabel: "Tags",
+};
+
+/**
+ * Tag group labels can also be placed on the side of the tag group. Side labels are most useful when vertical space is limited.
+ */
+export const SideLabel = Template.bind({});
+SideLabel.storyName = "Label position - side";
+SideLabel.tags = ["!dev"];
+SideLabel.parameters = {
+	chromatic: {
+		disableSnapshot: true,
+	},
+};
+SideLabel.args = {
+	fieldLabelPosition: "side",
+	items: exampleTagItems,
+	fieldLabel: "Tags",
+	helpText: "These tags were automatically added."
+};
+
 /**
  * A tag group can contain removable tags when the context is for editing or non-removable tags when tags are read-only. Removable and non-removable tags cannot be combined within the tag group.
+ *
+ * When horizontal space is limited in a tag group, the tags wrap to form another line. Individual tags don't wrap between lines; they'll either move to the next line or the text within the tag will truncate.
  */
-export const Removable = Template.bind({});
-Removable.tags = ["!dev"];
-Removable.parameters = {
+export const RemovableAndWrapping = Template.bind({});
+RemovableAndWrapping.storyName = "Removable and wrapping";
+RemovableAndWrapping.tags = ["!dev"];
+RemovableAndWrapping.parameters = {
 	chromatic: {
 		disableSnapshot: true,
 	},
 };
-Removable.args = {
+RemovableAndWrapping.args = {
+	fieldLabel: "Tags",
+	actionButtonText: "",
+	helpText: "",
 	isRemovable: true,
-	isEmphasized: false,
 	customStyles: {"max-width": "300px"},
```

Comment:
I don't think this is blocking anything, but do we want to update this to `max-inline-size` at all? Should we also mention that "implementations can set a max inline size on `.spectrum-TagGroup` to facilitate wrapping?" 

I don't think either are technically blocking.